### ISSUE TITLE
Determine Mobile App strategy

### ISSUE THREAD

**Issue by elmo** [elmo/upstairs#83]
From Michael:

But that also brings up the question of whether building the Upstairs app is important before launching the website. And also whether we need to agree on feature lists for both up front, because knowing what we’re going to want the app to do impacts what tools and features we train people to use here when they first encounter the website.

**Comment by elmo** [comment]
I agree the mobile app is a near essential element.  Unfortunately, it's the hardest thing to complete.  Especially if we're considering android and ios.   Either we'll have to pay someone to do it, or, I'll need to take a month of from work to get it done.   Both of these options are going to be difficult at our current state of funding.

**Comment by reviser273** [comment]
Katrina may be working to get these built by specialists. She can update.

**Comment by luvmodern** [comment]
Yes, I think we need to do this. We'll make a plan for mobile/tablet and desktop and build all at once.
